app/assets/javascripts/Components/repo_browser.jsx

```diff
@@ -168,15 +191,26 @@ class ManualCollectionForm extends React.Component {
               {I18n.t("submissions.collect.apply_late_penalty")}
             </label>
           </p>
-          <button
-            type="submit"
-            name="commit"
-            onClick={event => {
```

Comment:
I moved this out of the `onClick` logic and into the `onSubmit` logic for the form so I could test this properly, otherwise there were errors about a lack of implementation for `onSubmit`